deploy/deploy.sh

```diff
@@ -80,6 +78,8 @@ oc create secret generic rhsso-secret --from-literal=RHSSO_PWD=$RHSSO_PWD --from
 
 # Deploy 3scale Resources
 THREE_SCALE_URL=$(${DIR}/3scale.sh deploy)
+echo "Waiting for the ${THREE_SCALE_URL} to be reachable"
+wait_for "curl -s -o /dev/null -I -w '%{http_code}' ${THREE_SCALE_URL} | grep  -q 200" "Wait until api available" "10m" "120"
```

Comment:
If we are waiting for 3scale to be ready here, then I don't think we need to wait for another 2 minutes at the end: L109. we can probably remove that.